systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesIsolatedST.java

```diff
@@ -344,4 +347,75 @@ public void testKRaftMode(ExtensionContext extensionContext) {
         LOGGER.info("Waiting for clients to finish sending/receiving messages.");
         ClientUtils.waitForClientsSuccess(producerName, consumerName, INFRA_NAMESPACE, MESSAGE_COUNT);
     }
+    @IsolatedTest
+    void testSwitchingConnectStabilityIdentifiesFeatureGateOnAndOff(ExtensionContext extensionContext) {
+        assumeFalse(Environment.isOlmInstall() || Environment.isHelmInstall());
+
+        final TestStorage testStorage = new TestStorage(extensionContext);
+        final int connectReplicas = 1;
+        final int messageCount = 500;
+        List<EnvVar> coEnvVars = new ArrayList<>();
+
+        coEnvVars.add(new EnvVar(Environment.STRIMZI_FEATURE_GATES_ENV, "", null));
+
+        LOGGER.info("Deploying CO without CSI");
```

Comment:
CSI? Container Storage Interface? I guess it means something else here, so maybe you can use a full name in the log?